gammapy/data/tests/test_data_store.py

```diff
@@ -58,6 +59,15 @@ def test_DataStore_load(data_manager):
     events = data_store.load(obs_id=23037, filetype='events')
     assert isinstance(events, EventList)
 
+#@pytest.mark.xfail
```

Comment:
Looks like this is working now?
If so, please remove the comment line `#@pytest.mark.xfail`.